<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page100</class>
 <widget class="QWidget" name="page100">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>page100</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page100
{
background-color: rgb(170, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="label_HB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Heartbeat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMlabel" name="atcmLabel_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0x1234</string>
   </property>
   <property name="toolTip">
    <string>PLC_UPTIME_s</string>
   </property>
   <property name="whatsThis">
    <string>ATCMlabel widget</string>
   </property>
   <property name="variable">
    <string>PLC_UPTIME_s</string>
   </property>
   <property name="refresh">
    <number>500</number>
   </property>
   <property name="max">
    <string>0</string>
   </property>
   <property name="min">
    <string>0</string>
   </property>
   <property name="viewStatus">
    <bool>true</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="format">
    <enum>ATCMlabel::Hex</enum>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_menu">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string>menu</string>
   </property>
   <property name="statusvar">
    <string/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="refresh">
    <number>200</number>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>85</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="text">
    <string>TP1043_02_A</string>
   </property>
   <property name="pressText">
    <string>TP1043_02_A</string>
   </property>
   <property name="borderWidth">
    <number>1</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ATCMtime" name="atcmTime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>66</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>ATCM QTime</string>
   </property>
   <property name="whatsThis">
    <string>ATCMtime widget</string>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>255</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
  </widget>
  <widget class="ATCMdate" name="atcmDate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>32</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>ATCM QDate</string>
   </property>
   <property name="whatsThis">
    <string>ATCMdate widget</string>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>255</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="format">
    <enum>ATCMdate::YYYY_MM_DD</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>34</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmMQTT_Start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>120</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>5</pointsize>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string/>
   </property>
   <property name="passwordVar">
    <string/>
   </property>
   <property name="statusvar">
    <string/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="bgPressColor">
    <color>
     <red>255</red>
     <green>170</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="systemicons.qrc">
     <normaloff>:/systemicons/Play.png</normaloff>:/systemicons/Play.png</iconset>
   </property>
   <property name="pressIcon">
    <iconset resource="systemicons.qrc">
     <normaloff>:/systemicons/Stop_sign.png</normaloff>:/systemicons/Stop_sign.png</iconset>
   </property>
   <property name="borderWidth">
    <number>1</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ATCMcombobox" name="atcmComboBox_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>34</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>MQTT_Status</string>
   </property>
   <property name="whatsThis">
    <string>ATCMcombobox widget</string>
   </property>
   <property name="variable">
    <string>MQTT_Status</string>
   </property>
   <property name="refresh">
    <number>500</number>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="writeAcknowledge">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="mapping">
    <string>1;NOT_CONNECTED;2;CONNECTED;3;CONNECTED_WITH_ERRORS;4;DEVICE_BLACKLIST;5;NO_HOPE;0;ZERO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Broker">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>68</y>
     <width>84</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Broker</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBroker">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>68</y>
     <width>240</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>172.16.2.78</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTopic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>240</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>test_topic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Topic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Send Topic</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Msg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>134</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Sent Msg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSentMessage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>134</y>
     <width>360</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMlabel" name="atcmLabel_10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4660</string>
   </property>
   <property name="toolTip">
    <string>PLC_UPTIME_s</string>
   </property>
   <property name="whatsThis">
    <string>ATCMlabel widget</string>
   </property>
   <property name="variable">
    <string>PLC_UPTIME_s</string>
   </property>
   <property name="refresh">
    <number>500</number>
   </property>
   <property name="max">
    <string>0</string>
   </property>
   <property name="min">
    <string>0</string>
   </property>
   <property name="viewStatus">
    <bool>true</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="format">
    <enum>ATCMlabel::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_mid">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>168</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt mid</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMlabel" name="atcmLabel_11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>168</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="toolTip">
    <string>MQTT_mid</string>
   </property>
   <property name="whatsThis">
    <string>ATCMlabel widget</string>
   </property>
   <property name="variable">
    <string>MQTT_mid</string>
   </property>
   <property name="refresh">
    <number>500</number>
   </property>
   <property name="max">
    <string>0</string>
   </property>
   <property name="min">
    <string>0</string>
   </property>
   <property name="viewStatus">
    <bool>true</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="format">
    <enum>ATCMlabel::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Cnt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>168</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Msg Counter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Sent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>168</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubTopic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>202</y>
     <width>240</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>subscribed_topic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Topic_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>202</y>
     <width>84</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Topic</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Received">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>202</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblReceivedMsg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>237</y>
     <width>360</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MQTT_Msg_R">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>237</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mqtt Rec.Msg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMled" name="atcmLed">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>202</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>MQTT_ActiveTopic</string>
   </property>
   <property name="whatsThis">
    <string>ATCMled widget</string>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="variable">
    <string>MQTT_ActiveTopic</string>
   </property>
   <property name="refresh">
    <number>200</number>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="onIcon">
    <iconset>
     <normaloff>:/on.png</normaloff>:/on.png</iconset>
   </property>
   <property name="offIcon">
    <iconset>
     <normaloff>:/off.png</normaloff>:/off.png</iconset>
   </property>
  </widget>
  <widget class="ATCMled" name="atcmLed_2">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>68</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>MQTT_Broker</string>
   </property>
   <property name="whatsThis">
    <string>ATCMled widget</string>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="variable">
    <string>MQTT_Broker</string>
   </property>
   <property name="refresh">
    <number>200</number>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="onIcon">
    <iconset>
     <normaloff>:/on.png</normaloff>:/on.png</iconset>
   </property>
   <property name="offIcon">
    <iconset>
     <normaloff>:/off.png</normaloff>:/off.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMcombobox</class>
   <extends>QComboBox</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMdate</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMlabel</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMled</class>
   <extends>QLabel</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMtime</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="systemicons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>atcmButton_menu</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
